src/general/OptionsMenu.tscn

```diff
@@ -373,6 +376,72 @@ min_value = 10.0
 max_value = 50.0
 value = 20.0
 
+[node name="HBoxContainer6" type="HBoxContainer" parent="CenterContainer/VBoxContainer/PanelContainer/MarginContainer/Graphics/VBoxContainer"]
+margin_top = 261.0
+margin_right = 648.0
+margin_bottom = 283.0
+
+[node name="Label" type="Label" parent="CenterContainer/VBoxContainer/PanelContainer/MarginContainer/Graphics/VBoxContainer/HBoxContainer6"]
+margin_right = 42.0
+margin_bottom = 22.0
+text = "GPU:"
```

Comment:
These texts should be localized, see: [working_with_translations.md](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/working_with_translations.md#working-with-translation).